controlplane/rosa/controllers/rosacontrolplane_controller.go

```diff
@@ -350,6 +358,17 @@ func (r *ROSAControlPlaneReconciler) reconcileNormal(ctx context.Context, rosaSc
 		BillingAccountID(*rosaScope.Identity.Account).
 		SubnetIDs(rosaScope.ControlPlane.Spec.Subnets...).
 		STS(stsBuilder)
+
+	// specify tags
+	if len(rosaScope.ControlPlane.Spec.Tags) > 0 {
+		awsBuilder = awsBuilder.Tags(rosaScope.ControlPlane.Spec.Tags)
+	}
+
+	// etcd encryption kms key arn
+	if rosaScope.ControlPlane.Spec.EtcdEncryptionKMSArn != "" {
```

Comment:
Add check if the etcdEncryption is true

Reply:
Currently if EtcdEncryptionKMSArn is set, the etcdEncryption is overridded to be  true. If the overriding is removed, we will need to add the check